packages/kit-bg/src/services/ServiceScanQRCode/utils/parseQRCode/handlers/lightningNetwork.ts

```
@@ -0,0 +1,33 @@
+import { getNetworkIdsMap } from '@onekeyhq/shared/src/config/networkIds';
+import { EQRCodeHandlerType } from '@onekeyhq/shared/types/qrCode';
+
+import type { ILightningNetworkValue, IQRCodeHandler } from '../type';
+
+const lightningNetwork: IQRCodeHandler<ILightningNetworkValue> = async (
+  value,
+  options,
+) => {
+  if (value && value.startsWith('lnurl')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)